Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/DriverLocation.hs

```diff
@@ -28,6 +29,7 @@ data DriverLocation = DriverLocation
     coordinatesCalculatedAt :: UTCTime,
     createdAt :: UTCTime,
     updatedAt :: UTCTime,
-    merchantId :: Id DMerchant.Merchant
+    merchantId :: Id DMerchant.Merchant,
+    merchantOperatingCityId :: Maybe (Id DMOC.MerchantOperatingCity) -- Need to check that location service send mocid
```

Comment:
I think here changes in LTS repo required to make it work, right @hkmangla @khuzema786  ?